client/blocks/nav-unification-modal/index.jsx

```diff
@@ -47,12 +48,17 @@ const Page = ( { heading, content, image } ) => {
 
 const Modal = () => {
 	const dispatch = useDispatch();
+	const isNewUser = useSelector( isNavUnificationNewUser );
 	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
 	const hasPreferences = useSelector( hasReceivedRemotePreferences );
 	const dismissPreference = `nav-unification-modal-${ userId }`;
 	const isDismissed = useSelector( ( state ) => getPreference( state, dismissPreference ) );
 	const translate = useTranslate();
 
+	if ( isNewUser ) {
```

Comment:
```suggestion
    // Don't show Modal to new users as they aren't aware of any other type of Navigation
	if ( isNewUser ) {
```

Reply:
I do wonder whether the small number of users who registered after 5th March but before launch date will have a nasty surprise when the whole interface changes with no explanation.

Reply:
Is there a way to enable the modal for users who registered prior to launch day?

Reply:
Oh, good point, didn't realize about that. With this new user ID threshold, users who registered between 5th March and 10th March will get nav unification without the modal. 

I can think of two options:
- We update the threshold to the ID of the last user who registered today. That way we mitigate the number of users who will fall into this non-ideal scenario.
- We add a date check for showing the modal, so only users who signed up after 10th March at 9am UTC see the modal.

Reply:
I would prefer 
> We update the threshold to the ID of the last user who registered today. That way we mitigate the number of users who will fall into this non-ideal scenario.

so that we also more consistent on "who" is regarded new user.

Reply:
Noted it in the launch timeline post as a task to do prior merging this PR: paYJgx-1rb-p2

Reply:
Thanks!